components/ZilPay/zilpay-base.ts

```diff
@@ -149,7 +149,7 @@ export class ZilPayBase {
             let xInit = '0x2d7e1a96ac0592cd1ac2c58aa1662de6fe71c5b9'
 
             if (net === 'testnet') {
-                XWALLET = 'zil1qvyhwc7s3g5eayrkqf88qens9qgzxqwtd6nsje'
+                XWALLET = 'zil1nrxvlzmt4vyd5z2r3pklcg78hutez7kenq7lr3'
```

Comment:
Hi @devwistkey, why this change & the tiktok icons?

Reply:
Noted, I'll try to revert the xwallet. Maybe that happened when checkout branch @tralkan